recipes/medcoupling/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "medcoupling" %}
+{% set version = "9.10.0" %}
+{% set petsc_ver = "3.17.1" %}
+
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - fn: 'medcoupling_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/medcoupling.git;a=snapshot;h=fe2e38d301902c626f644907e00e499552bb2fa5;sf=tgz # 9.10
+    sha256: e727bf8b8bfe848a0e80605e269b8be818bebf9ee098cbb5d6168a920b210c46
+  - fn: 'config_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/configuration.git;a=snapshot;h=25f724f7a6c0000330a40c3851dcd8bc2493e1fa;sf=tgz # 9.10.0
+    folder: deps/config
+    sha256: 53cbdf2def53e32b2237e948b7dbe8e4051e69e96319a786cd51aba0c915703d
+
+
+build:
+  detect_binary_files_with_prefix: true
+  # string: aster_py{{ CONDA_PY }}_mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [mpi != "nompi"]
+  # string: aster_py{{ CONDA_PY }}_nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}           # [mpi == "nompi"]
+  number: 0
+  skip: true  # [py2k or osx or py>=312 or win]
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+    - make                                      # [not win]
+    - ninja                                     # [win]
+    - cmake
+    - swig >=4.0.2
+
+  host:
+    - python
+    - numpy >=1.22
```

```suggestion
    - numpy
```

Numpy will build with an "old numpy" to ensure forward compatibility